go.sum

```diff
@@ -539,6 +539,8 @@ github.com/go-morph/morph v0.0.0-20210713131459-1c24ffc64e79 h1:1vXBa7+ganiI20fT
 github.com/go-morph/morph v0.0.0-20210713131459-1c24ffc64e79/go.mod h1:XQh5WcM351wOV3z3zEWRM8RaJ65E2p4P7WWbmFAi8x4=
 github.com/go-morph/morph v0.0.0-20211202070306-dbdc0736c17e h1:iS/jL0Xij+6O5d/VXrt+VOZfATHCTDIjBVVthVpHFY0=
 github.com/go-morph/morph v0.0.0-20211202070306-dbdc0736c17e/go.mod h1:XQh5WcM351wOV3z3zEWRM8RaJ65E2p4P7WWbmFAi8x4=
+github.com/go-morph/morph v0.0.0-20220110203813-7e65a95885ea h1:l2xnsVP6YW9DZet6HfUhMKtEOPyGg2QsRyDPLECnW1o=
```

Comment:
Slightly unrelated but any reason why morph is not under the `mattermost` organization?

Reply:
@mgdelacroix might know :stuck_out_tongue:

Reply:
I don't know the actual reason to keep this in a separate organization but transferring the ownership to mattermost sounds good to me. I'll ping people to make this happen.

Reply:
It was born out of the org to allow people from the outside to contribute independently. 0/5 on what schema to follow from this point on